ecommerce-ui/src/components/RentalPage.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import AirBnbList from './AirBnbList'
+import ShoppingCart from './ShoppingCart'
+import './components.css';
+import airbnbs from '../airbnbs.json';
+
+class RentalPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedRentals: [],
+            allBnbs: airbnbs
+        };
+    }
```

Comment:
this.addRental = this.addRental.bind(this);
        this.removeRental = this.removeRental.bind(this);